frontend/src/pages/ActivityReport/Pages/Review/ReviewItem.js

```diff
@@ -16,6 +20,18 @@ const ReviewItem = ({ label, name, path }) => {
     values = values.map((v) => _.get(v, path));
   }
 
+  values = values.map((v) => {
+    // If not a valid url, then its most likely just text, so leave it as is
+    if (!isValidURL(v)) {
```

Comment:
the isValidURL implementation incorrectly marks any links to to other reports (or future parts of the smarthub) as not being links. Maybe https://developer.mozilla.org/en-US/docs/Web/API/URL/URL would be a better way to parse out URL parts.

Reply:
Yeah, ended up using that